docs/modules/ROOT/partials/getting-started/ref-registry-security-configuration.adoc

```diff
@@ -120,7 +120,7 @@ You can set the following options to `true` to enable role-based authorization i
 |Boolean
 |`false`
 |`ROLE_BASED_AUTHZ_ENABLED`
```

Comment:
These should be updated since you've removed the explicit env. vars, e.g. `APICURIO_AUTH_ROLE_BASED_AUTORIZATION`. Or, we link the users to the [mapping rules](https://github.com/eclipse/microprofile-config/blob/main/spec/src/main/asciidoc/configsources.asciidoc#environment-variables-mapping-rules) and only mention the raw property.

Reply:
Did you copy/paste this value or type it in to this comment?  `APICURIO_AUTH_ROLE_BASED_AUTORIZATION`  I ask because of the typo in `AUTORIZATION`

Reply:
It's a typo.